build-tools/scripts/PrepareWindows.targets

```diff
@@ -18,6 +18,7 @@
     <MSBuild Projects="$(MSBuildThisFileDirectory)..\download-bundle\download-bundle.csproj" />
     <Exec Command="$(_NuGet) restore Xamarin.Android.sln" WorkingDirectory="$(_TopDir)" />
     <Exec Command="$(_NuGet) restore Xamarin.Android-Tests.sln" WorkingDirectory="$(_TopDir)" />
+    <MSBuild Projects="Xamarin.Android-Tests.sln" Targets="Restore" />
```

Comment:
*This* is in keeping with what I suggested [earlier](https://github.com/xamarin/xamarin-android/pull/2474/files#r239052052) (great minds think alike), though I suspect that you *also* want to call the `Restore` target on `Xamarin.Android.sln` as well.